docs/admin.rst

```diff
@@ -294,6 +294,47 @@ web server to apply the changes:
 
   sudo service apache2 restart
 
+.. _submission-cleanup:
+
+Cleaning up deleted submissions
+&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+When submissions are deleted through the web interface, their database
+records are deleted and their encrypted files are securely wiped. For
+large files, secure removal can take some time, and it's possible,
+though unlikely, that it can be interrupted, for example by a server
+reboot. In older versions of SecureDrop this could leave a submission
+file present without a database record.
+
+As of SecureDrop 1.0.0, automated checks send OSSEC alerts when this
+situation is detected, recommending you run ``./manage.py
+list-disconnected-fs-submissions`` to see the files affected. (As with
+any ``manage.py`` usage, you would first SSH to the application
+server, then run ``sudo -u www-data bash`` and ``cd
+/var/www/securedrop``.) You then have the option of running
+``./manage.py delete-disconnected-fs-submissions`` to clean them up. As
+with any potentially destructive operation, it's recommended that you
+:doc:`back the system up <backup_and_restore>` before doing so.
+
+There is also the inverse scenario, where a database record could
+point to a file that no longer exists. This would usually only have
+happened as a result of disaster recovery, where perhaps the database
+was recovered from a failed hard drive, but some submissions could not
+be. The OSSEC alert in this case would recommend running ``./manage.py
+list-disconnected-db-submissions``. To clean up the affected records
+you would run ``./manage.py
+delete-disconnected-db-submissions``. Again, a :doc:`backup
+<backup_and_restore>` immediately beforehand would be prudent.
+
+Even when submissions are completely removed from the application
+server, their encrypted files may still exist in backups. Because the
+private key for decrypting them (part of the :ref:`Submission Key
```

Comment:
Another issue here is  exposed metadata about submission times etc. So it might be good to leave out the low risk part and just encourage careful handling of backups.